.gitignore

```diff
@@ -0,0 +1,3 @@
+volumes/public/apps/1/*
```

Comment:
Thank you for this ---I'd meant to do this! We could actually ignore everything in apps, or even public